LondonBritishLibrary/orient/BLorient822.xml

```diff
@@ -31,80 +32,163 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                            <note>The sixth chapter begins with <persName ref="PRS13241Archelaus">Archelaus</persName>,
+                              the son of <persName ref="PRS5299Herod">Herod</persName>,
+                              and ends with John the Galilean</note>
+                              <msItem xml:id="ms_i1.6.1">
+                                <locus from="90r"></locus>
+                                <title type="complete" xml:lang="en">Unit regarding
+                                  <persName ref="PRS5741Johnthe">St John the Baptist</persName>.</title>
+                                <textLang mainLang="gez"></textLang>
+                                <incipit xml:lang="gez">ወዓዲ፡ ቀተሎ፡ ለዮሐኖን፡ ወልደ፡ ዘካርያስ፡ ካህን፡ እስመ፡ ውእቱ፡
+                                ዘለፎ፡ እንዘ፡ ይብል፡ ኢይደልወከ፡ ትንሣእ፡ ብእሲተ፡ እኁከ፡ እንዘ፡ ሀሎ፡ ሕያወ፡ ወወለደ፡ እምኔሃ፡
+                              ክልኤ፡ ውሉደ፡ ወዮሐኖንሰዝ፡ ውእቱ፡ ዘይሰመይ፡ ይሕያ፡ ወልደ፡ ዝካርያ፡ ወበዘመነ፡ አንጥፊስ፡ ወልደ፡
+                            ሄሮድስ፡ ሞተ፡ አውግስጦስ፡ ቄሳር፡ ወነግሠ፡ ጤባርዮስ፡ ቄሳር፡ ወኮነ፡ ብእሴ፡ እኩየ፡ ምግባር፡</incipit>
+                              </msItem>
                         </msItem>
                         <msItem xml:id="ms_i1.7">
                             <locus from="103v" to="122v"/>
-                            <title type="complete" ref="LIT2598Yosipp#Kefl7"> narrates the events of the siege of Jerusalem by Titus. 
-                            </title>
+                            <title type="complete" ref="LIT2598Yosipp#Kefl7"></title>
+                            <textLang mainLang="gez"></textLang>
+                            <note>The seventh chapter narrates the events of the siege
+                              of <placeName ref="LOC3957Jerusa">Jerusalem</placeName> by
+                              <persName ref="PRS9533Titus">Titus</persName>.</note>
                         </msItem>
                         <msItem xml:id="ms_i1.8">
                             <locus from="122v" to="136r"/>
-                            <title type="complete" ref="LIT2598Yosipp#Kefl8"> brings the history of the siege of its close. 
-                            </title>
-                            <colophon xml:id="coloph1" xml:lang="gez">
-                                <locus target="#136r"/>
-                                ተፈጸመ፡ መጽሐፈ፡ ዮሴፍ፡ ወልደ፡ ኮርዩን፡ ካህን፡ ዘያየድዕ፡ ዜና፡ አይሁድ፡ ውስተ፡
-                                ቤት፡ ዳግማዊ፡ ወዝክረ፡ ነገሥቶሙ፡ ወነገርኒ፡ ዘኮነ፡ በመዋዕሊሆሙ፡ እምአሜ፡ ሕንጸተ፡ 
-                                ቤት፡ ዳግማዊ፡ ወግብአቶሙ፡ ኀቤሁ፡ እስከ፡ ተነሥተ፡ ወተፄወዉ፡ እምኔሁ። ስብሐት፡ ወአኰቴት፡ etc</colophon>
-                            
+                            <title type="complete" ref="LIT2598Yosipp#Kefl8"></title>
+                            <textLang mainLang="gez"></textLang>
+                            <note>The eighth chapter brings the history of the siege of its close.</note>
+
+
+                        </msItem>
+                        <colophon xml:id="coloph1" xml:lang="gez">
```

Comment:
Is this rather a colophon of the work?

Reply:
I think it is, it is also nested in the first ms_item, if I am not wrong.

Reply:
Does it have an xml:id in the work record? I think it would be preferable to encode it in the manuscript transcription, or make explicit in any other way that it is not particular to this manuscript.

Reply:
Sorry, you are right! the Colophon has been added to the work record as well